prime-router/src/test/kotlin/ReportTests.kt

```diff
@@ -32,6 +42,11 @@ class ReportTests {
 
     val rcvr = Receiver("name", "org", Topic.TEST, CustomerStatus.INACTIVE, "schema", Report.Format.CSV)
 
+    val azuriteContainer =
```

Comment:
As an FYI, I had missed this earlier, but there are nice annotations `@Container` and `@Testcontainers` which handles starting teh container automatically.

Reply:
Ya I saw BlobAccessTests made use of it but my tests fail if I update them to that and I run the class. If I run the individual tests they work then.